policyengine_us/parameters/gov/states/dc/tax/income/credits/eitc/no_children/phase_out/start.yaml

```diff
@@ -0,0 +1,9 @@
+description: DC EITC Without Qualifying Children AGI Floor
+values:
+  2022-01-01: 20_532
+metadata:
+  unit: USD
+  label: DC EITC Without Qualifying Children AGI Floor
+  reference:
+    - title: Code of the District of Columbia § 47–1806.04 (f)(1)(C)(ii)
```

Comment:
Please cite the tax form as well, using an additional title/href bullet, since that will have the 2022 value

Reply:
Added the tax form citation